oojs.md

```diff
@@ -7,15 +7,27 @@ Define a `Playlist` class. It should have...
 - An `addSong` (Function) method that adds a song to the `songs` property
 
 ```js
-Your answer goes here
+
+var playlist = {
```

Comment:
`Playlist` needs to be a class, not an object literal. Also, when we define classes the first letter is capitalized.